src/org/jitsi/impl/neomedia/transform/AbsSendTimeEngine.java

```diff
@@ -129,8 +130,66 @@ private void setTimestamp(byte[] buf, int off)
      * disable this transformer.
      * @param id the ID to set.
      */
-    public void setExtensionID(int id)
+    public void setExtensionID(byte id)
     {
         extensionID = id;
     }
+
+    /**
+     *
+     * @param packet is the RawPacket
+     * @returns the Absolute Send Time header extension if present
+     */
+    public HeaderExtension hasAbsoluteSendTimeExtension(RawPacket packet)
+    {
+
+        if(packet.getExtensionBit() )
+        {
+            return packet.getHeaderExtension(getAbsSendTimeExtensionID());
+        }
+        return null;
+    }
+
+    /**
+     * @ToDo findout the prefered way of setting {@Link extensionID}
+     * returns AbsSendTime Extension ID
+     * @return
+     */
+    public byte getAbsSendTimeExtensionID(){
+        return extensionID == -1 ? extensionID_for_debugging : extensionID ;
```

Comment:
Can we please remove extensionID_for_debugging? When testing with jitsi-meet abs-send-time is configured with id=3, so you can already test it.